srcpkgs/i3lock-color/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'i3lock-color'
+pkgname=i3lock-color
+version=2.11
+revision=1
+wrksrc="${pkgname}-${version}-c"
+build_style=gnu-configure
+hostmakedepends="pkg-config automake"
+makedepends="libev-devel cairo-devel pam-devel libxkbcommon-devel xcb-util-devel
+	xcb-util-image-devel libjpeg-turbo-devel"
```

Comment:
should be one whitespace, not 1 tab.

see template with multiple lines like gtk+3 to see how it is done.